cf-agent/verify_files.c

```diff
@@ -650,33 +649,33 @@ static PromiseResult RenderTemplateMustache(EvalContext *ctx, const Promise *pp,
     Buffer *output_buffer = BufferNew();
 
     char *message;
-    if ( strcmp("inline_mustache", a.template_method) == 0)
+    if ( strcmp("inline_mustache", attr->template_method) == 0)
```

Comment:
Might as well fix the extra space before `strcmp` here.

Reply:
But then I might as well change it to `StringEqual()`, and then:

![MindlessBelovedAmericanlobster-size_restricted](https://user-images.githubusercontent.com/4048546/85017621-0bc0f200-b16c-11ea-95fd-291e17c6d9c9.gif)

Reply:
Removed the space.